README.md

```diff
@@ -16,7 +16,7 @@ Features include:
 - caching and reusing build results
 - efficient use of disk space due to copy-on-write semantics
 - a simple JSON API for interacting with the service
-- ([wip](https://github.com/skroutz/mistry/pull/17)) a web view for inspecting the progress and result of builds
+- a web view for inspecting the progress and result of builds
```

Comment:
This needs a rebase. I slightly updated this point.

Reply:
True, fixed.